server/hot-reloader.js

```diff
@@ -1,6 +1,7 @@
 import { join, relative, sep } from 'path'
 import webpackDevMiddleware from 'webpack-dev-middleware'
 import webpackHotMiddleware from 'webpack-hot-middleware'
+import isWindowsBash from 'is-windows-bash'
```

Comment:
Oh whoops, totally forgot to check that in. Will update my package to support ES5 as well